tests/modules/nf-core/proteus/readproteingroups/delete_me/MaxQuant_contrasts.csv

```diff
@@ -0,0 +1,5 @@
+id,variable,reference,target,blocking
```

Comment:
test files should be in the test dataset repo

Reply:
Opp. They are already, I just forgot to delete the folder :'D